viewproduct.php

```diff
@@ -10,162 +10,157 @@
+        <?= '<input type="hidden" name="productid" value="' . $data['ProductID'] . '" />' ?>
+        <img src="<?= $data['ProductImage'] ?>" width="270" height="270" alt="<?= $data['ProductName'] ?>">
+      </div>
+      <div class="col_3">
+        <div class="product_description">
+          <span><?= $data['ProductBrand'] ?></span>
+          <h3><?= $data['ProductName'] ?></h3>
+          <hr>
+          <p>Design type: <?= $data['ProductDesignType'] ?></p>
+          <p>Bike type: <?= $data['ProductBikeType'] ?></p>
+          <p>Availability: <?= $data['ProductAvailability'] ?></p>
+          <p>Discount: <?= $data['ProductDiscountAmount'] ?>%</p>
+        </div>
+        <div class="product_note">Note: This item is a universal custom fit component, and is not a direct fitment for your machine. These parts may require modification and we recommend installation by a certified mechanic. Please have machine specifications ready when you call.</div>
+        <p>Quantity: <input type="number" value="1" name="quantity" min="1" step="1" class="input_quantity" required></p>
+        <div class="product_price">
+          <span>LKR:<?= $data['ProductPrice'] ?></span>
+          <a href="cart.php"><input name="addtocart" type="submit" value="Add to cart" class="cart_btn"></a>
+        </div>
+        <span><s><?= $data['InitialProductPrice'] ?></s></span>
+      </div>
+    <?php } } else {?>
+      <h3>Oops! Product Unavailable!</h3>
+  <?php } ?>
+</form>
+<form method="post">
+    <table>
+        <tr>
+            <th><b>Product Reviews</b></th>
+        </tr>
+        <input type="hidden" name="productid" value="<?= $PRODUCTID ?>">
+        <?php if ($no_of_reviews > 0): ?>
+            <?php while ($data = mysqli_fetch_array($result_reviews)): ?>
+                <tr>
+                    <td><b>Reviewed On: </b><?= $data['ReviewDate'] ?></td>
+                </tr>
+                <tr>
+                    <td><b>Reviewed By: </b><?= $data['Username'] ?></td>
+                </tr>
+                <tr>
+                    <td><b>Review: </b><?= $data['Review'] ?></td>
+                </tr>
+                <tr>
+                    <td><hr color="#00CCCC"></td>
+                </tr>
+            <?php endwhile ?>
+        <?php else: ?>
+            <tr>
+                <td>'0' Reviews for this product</td>
+            </tr>
+        <?php endif ?>
+    </table>
+    <br>
+    <?php if(isset($_SESSION['username'])): ?>
+        <table>
+            <tr>
+                <th>Leave a Review</th>
+            </tr>
+            <tr>
+                <td><b>Review*</b> <textarea name="review" required></textarea></td>
+            </tr>
+        </table>
+        <table>
+            <tr>
+                <td><?= $MESSAGE_REVIEW ?></td>
+                <td colspan="6" class="tf"><input type="submit" name="addreview" class="cart_btn" value="Add Review"></td>
+            </tr>
+        </table>
+    <?php endif ?>
+</form>
```

Comment:
- Use alternative syntax for control structures (if, while, foreach) to make the code easier to read.
- Avoid mixing HTML and PHP code in the same block. 
- Use prepared statements and parameterized queries to prevent SQL injection attacks.